lib/Field/FieldRepeatableControls.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { bool, number } from 'prop-types';
+import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
+import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
+
+export function FieldRepeatableControls({ repeatPosition, isLastRepeat }) {
+  const label = `${repeatPosition < 9 ? '0' : ''}${repeatPosition + 1}`;
+
+  return (
+    <div className="flex flex-row-reverse">
```

Comment:
Any particular reason for the `row-reverse`?

Reply:
Yes - this was to get the row to "float" to the right - is there a better way to do this?? I didn't want do use margin-left: auto, because then I'd need to swap it when the text dir is RTL - this seems to work magically with RTL.

Reply:
ah we can use `flex-end` to justify to the right instead 😅 knew there was a better way...